.travis.yml

```diff
@@ -15,7 +15,7 @@ install:
 script:
   - python all_tests.py --coverage
 after_success:
-  - coveralls
+  - coveralls --rcfile=.coveragerc
```

Comment:
I think we still have problems with the actual coverage being not precise. Looking at the files in coveralls.io, I think this report is more accurate for master:

https://coveralls.io/github/ocadotechnology/aimmo?branch=coveralls